journalbeat/docs/general-options.asciidoc

```diff
@@ -29,26 +29,28 @@ data path. See the <<directory-layout>> section for details. The default is `${p
 ----
 
 [float]
-==== `backoff`
+==== `backoff` deprecated[5.6.1,Use the option under `paths` instead.]
```

Comment:
I decided to use an inline deprecation rather than putting this on a separate line because it looks better (otherwise, the inline text is a very large font...too distracting to a minor deprecation like this).